### PR TITLE
Add coding in the open, CI/CD principles

### PR DESCRIPTION
**Pull request by jonodrew**
This PR adds content about how we store code and principles on continuous integration and continuous deployment.

I'm looking for feedback on:

- how close to reality this is: there's no point in saying "We do things this way" if we don't and we don't intend to.
- location: is this the right place in the taxonomy for this content?

### CODE REVIEW

File: source/principles/general/index.html.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+out more generic principles for technical folk at the department.
+
+## Make things open, it makes things better
+All of the code that we produce is classified at OFFICIAL and is paid for by the taxpayer. We have a duty to make that 
+code open, to encourage reuse, and in doing so reduce waste. That's why we use GitHub to store our code, unless there's 
+a security or policy reason to keep our work secret. Even the spooky folks at the Government Communications Headquarters
+(GCHQ) have published a number of [open-source repos](https://github.com/gchq). [We've written guidance on how to use our 
+preferred tool, GitHub](../../standards/storing-source-code/index.html.md.erb)
+
+## Continuous integration
```
Comment (saliceti) [inline source/principles/general/index.html.md:19]: This overlaps with https://technical-guidance.education.gov.uk/guides/continuous-delivery/
Could we link to it as least?

File: source/principles/index.html.md.erb [hunk 2]
```diff
@@ -5,4 +5,4 @@ weight: 20
 
 # <%= current_page.data.title %>
 
-See sidebar to navigate principles.
+See sidebar to navigate principles.
```
Comment (saliceti) [inline source/principles/index.html.md.erb:8]: Was the EOF line break removed by mistake?

File: source/principles/general/index.html.md (hunk 1 above)
Comment (jonodrew) [reply source/principles/general/index.html.md:19]: Yeah, good shout!

File: source/principles/index.html.md.erb (hunk 2 above)
Comment (jonodrew) [reply source/principles/index.html.md.erb:8]: Good catch thank you

**Review by saliceti — APPROVED** [review]

**Pull request merged**